src/background/services/background.ts

```diff
@@ -61,6 +62,30 @@ export class Background {
     this.sendToPopup.start();
   }
 
+  async injectPolyfill() {
+    try {
+      // TODO: When Firefox 128 is old enough, inject directly via manifest.
+      // Also see: injectPolyfill in contentScript
```

Comment:
Can you please log an issue for this? Please add the link for the issue in the comment as well.